tests/pipelines/employees.py

```diff
@@ -92,10 +147,34 @@ class Transformer(Phase):
     steps = [
         combine_full_name,
         calculate_annual_salary,
-        calculate_bonus_percent
+        calculate_bonus_percent,
```

Comment:
I'm hoping to keep the code sample in the readme.md consistent with this actually-useful actually-tested code - not all of it, but I was thinking the phases and pipeline.  But that can wait until this settles